src/index.ts

```diff
@@ -70,13 +81,35 @@ function emitDom() {
 
         Object.keys(descriptions).forEach(name => {
             idl.interfaces!.interface[name] = {
-                comment: descriptions[name],
+                comment: transformVerbosity(name, descriptions[name]),
             } as Browser.Interface;
         });
 
         return idl;
     }
 
+    function transformVerbosity(name: string, description: string): string {
+        for (const regTemplate of removeVerboseIntroductions) {
+            let template: string, replace: string;
+            if (Array.isArray(regTemplate)) {
+                template = regTemplate[0].source;
+                replace = regTemplate[1];
+            }
+            else {
+                template = regTemplate.source;
+                replace = '';
+            }
+
+            const reg = new RegExp(template.replace(/\$\{name\}/g, name).replace(/\s+/, '\\s*') + '\\s*', 'i');
```

Comment:
Hm. Does the `.replace(/\s+/, '\\s*')` turn out to be necessary? That's a little strange.

Reply:
Over-cautious against double-spaces. I'll remove and recheck.